packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -74,11 +77,11 @@ export class ApexTestRunner {
             ),
             10
           ) - 1; // Remove one because vscode location is zero based
-        channelService.appendLine('-----------------------------------------');
-        channelService.appendLine(stackTrace);
-        channelService.appendLine(errorMessage);
-        channelService.appendLine('-----------------------------------------');
-        channelService.showChannelOutput();
+        OUTPUT_CHANNEL.appendLine('-----------------------------------------');
+        OUTPUT_CHANNEL.appendLine(stackTrace);
+        OUTPUT_CHANNEL.appendLine(errorMessage);
+        OUTPUT_CHANNEL.appendLine('-----------------------------------------');
+        OUTPUT_CHANNEL.show(true);
```

Comment:
Fixed the test sidebar outputting to Salesforce CLI when you click on a failure as well.